### PR TITLE
Create scaffold / no-operation classes

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

**Review by Alicja-Kainos — DISMISSED** [review]
Looks fine so far, if CI pass, I approve it :)

File: inbound/supplierincomingmq/incoming_mq.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+class SupplierIncomingMQ:
```
Comment (Alicja-Kainos) [inline inbound/supplierincomingmq/incoming_mq.py]: @Broom94, please rename the package 'supplierincomingmq' into 'supplier_incoming_mq'

File: inbound/request/inbound_handler.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from utilities import integration_adaptors_logger as log, timing
+
+from inbound.supplierincomingmq.incoming_mq import SupplierIncomingMQ
+from edifact.incoming.edifact_to_fhir import EdifactToFhir
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+class Inbound_Handler():
```
Comment (mattd-kainos) [inline inbound/request/inbound_handler.py]: Don't use underscore in python class names

File: inbound/supplierincomingmq/incoming_mq.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+class Supplier_Incoming_MQ:
```
Comment (mattd-kainos) [inline inbound/supplierincomingmq/incoming_mq.py]: Inbound instead of Incoming for consistency

**Review by mattd-kainos — CHANGES_REQUESTED** [review]
Should consider stubbing out more of the component document at https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/2068152352/Adaptor+Development+Diagrams

File: inbound/request/inbound_handler.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        self.edifact_to_fhir = EdifactToFhir()
+        self.supplier_incoming_mq = SupplierInboundMQ()
+        self.inbound_sequence_number_manager = InboundSequenceNumberManager()
+        self.edifact_recep_producer = EdifactRecepProducer()
+        self.edifact_recep_consumer = EdifactRecepConsumer()
+
+    async def on_message_recieved(self, edifact_message):
+        if edifact_message.message:
+            fhir_message = self.edifact_to_fhir.convert_edifact_to_fhir(edifact_message)
+            if edifact_message.sequence_number:
```
Comment (mattd-kainos) [inline inbound/request/inbound_handler.py]: no if/else here these three always happen

File: inbound/supplierinboundmq/inbound_mq.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (Alicja-Kainos) [inline inbound/supplierinboundmq/inbound_mq.py]: Screenshot 2020-05-19 at 09 57 39
supplier_inbound_mq ? look at the package name, please

File: edifact/incoming/edifact_recep_consumer.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+
+class EdifactRecepConsumer:
```
Comment (mattd-kainos) [inline edifact/incoming/edifact_recep_consumer.py]: Just spotted this, could these three mirror the outbound structure, so put the files in inbound/converter

These are mainly orchestration classes. The /edifact package contains internal representations of edifact message segments and is used by the whole application.

Comment (mattd-kainos) [reply edifact/incoming/edifact_recep_consumer.py]: We should probably do a breakdown of the package structure to explain what goes where and why...

Comment (Broom94) [reply edifact/incoming/edifact_recep_consumer.py]: By three do you mean inbound sequence number manager, Edifact Recep Consumer and edifact recep producer?

Comment (mattd-kainos) [reply edifact/incoming/edifact_recep_consumer.py]: EdifactRecepConsumer, EdifactRecepProducer, EdifactToFhir

**Review by mattd-kainos — APPROVED** [review]

**Review by Alicja-Kainos — APPROVED** [review]

**Pull request merged**